etc/check_license.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+year=$(date +"%Y")
+copyright=$"// Copyright (C) MongoDB, Inc. $year-present.
```

Comment:
I just realize we don't update the year on all files. [mongo/client.go](https://github.com/mongodb/mongo-go-driver/blob/master/mongo/client.go#LL1C1-L1C45) for instance uses 2017-present. Should the script update these to the current year?

Reply:
I believe that the most correct thing is to write either the current year or the year of the first commit when a file is created and then not change it. I wasn't able to find an answer to your exact question, but I referenced these articles to try to find an answer:
* https://www.linkedin.com/pulse/how-apply-apache-20-license-your-open-source-software-vladimír-gorej
* https://opensource.stackexchange.com/questions/6389/how-do-the-years-specified-in-a-copyright-statement-work